services/blockchain-indexer/tests/unit/shared/dataservice/business/mainchain.test.js

```diff
@@ -0,0 +1,172 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2023 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+const config = require('../../../../../config');
+
+const mockedFilePath = '../../../../../shared/dataService/business/network';
```

Comment:
Would it be better to use dirname here?

```suggestion
const mockedFilePath = ${__dirname}/../../../../../shared/dataService/business/network`;
```